index.html

```diff
@@ -3,10 +3,144 @@
+          <p class="home-location-info">@StudentsToWorkd Collage-EEUU</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="activities" id="activities">
+      <div class="activities-subtitle">
+        <h2> Main Program </h2>
+        <hr>
+      </div>
+      <div class="containerBoxs">
+        <div class="boxs">
+          <img src="/images/nuevo-mensaje.png" class="imagesActivity" alt="" srcset="">
+          <h4>Forum</h4>
+          <p>Enjoy taking some time Take the time to share your thoughts and opinions with experts for each topic.</p>
+        </div>
+        <div class="boxs">
+          <img src="/images/taller.png" class="imagesActivity" alt="" srcset="">
+          <h4>Advisory</h4>
+          <p>Get free advice and recommendations from the most experts in the field, negotiate according to your interests.</p>
+        </div>
+        <div class="boxs">
+          <img src="/images/comida.png" class="imagesActivity" alt="" srcset="">
+          <h4>Aperitif</h4>
+          <p>Enjoy a delicious appetizer while you enjoy observing the greatest works of great artists of the genre.</p>
+        </div>
+        <div class="boxs">
+          <img src="/images/tendencia.png" class="imagesActivity" alt="" srcset="">
+          <h4>Trends</h4>
+          <p>Find out about the latest trends, update your knowledge, get ideas, find best and most detailed information.</p>
+        </div>
+        <div class="boxs">
+          <img src="/images/exhibicion.png" class="imagesActivity" alt="" srcset="">
+          <h4>Exhibition</h4>
+          <p>You can enjoy a wide gallery of exhibitions, where you can find a very varied range of works made in different cultures.</p>
+        </div>  
+      </div>
+      <div class="containerButton">
+        <button type="button" class="Button" id="Button">Join Urban Art Festival</button>
+      </div>
+    </section>
+
+    <section class="DinamicsSection">
+      <div class="grid-section" id="artistsContainer">
+ 
+      </div>
+    </section>
+    
+    <section class="parthnerSection">
+      <h2>Partner</h2>
+      <div>
+        <ul class="parthner-conten">
+          <li class="parthnerLi"><img src="/images/parthner/Adobe-Logo-1.png" alt="" srcset=""></li>
+          <li class="parthnerLi"><img src="/images/parthner/RedBull2.png" alt="" srcset=""></li>
+          <li class="parthnerLi"><img src="/images/parthner/amnesty-international.png" alt="" srcset=""></li>
+          <li class="parthnerLi"><img src="/images/parthner/google.png" alt="" srcset=""></li>
+          <li class="parthnerLi"><img src="/images/parthner/patagonia.png" alt="" srcset=""></li>
+        </ul>
+      </div>
+    </section>
+      <div class="footer">
+        <img src="/images/LogoFer.png" alt="" srcset="">
+        <div class="licencedFooter">
+          <p>2023 Fernando Jose Montes Linares. Some Rights Reserved.</p>
+          <p>"ご訪問いただき、ありがとうございます。皆様にとって素晴らしい日々が訪れますように。"</p>
+        </div>
+      </div>
```

Comment:
- [ ] Could you please take a look at the "Partner" title? It seems that the font size, color, and formatting might need some adjustments to match the template. Additionally, if you could add the line below the title to give your project a closer resemblance to the Behance example, that would be great!

- [ ] Some of your partner logos are almost invicible, kindly try to use another ones.

- [ ] Also, I noticed that the footer content isn't centered and is currently aligned to the left side of your page. It would be wonderful if you could center the footer content for a more polished look. Your efforts to fine-tune these details will surely make your project shine! 🌟

   ### Your design:
   ![image](https://github.com/Fernando426/CapstoneN01/assets/107952724/2d7c45be-cf18-42b7-b531-ea265d11941d)

   ### Behance design:
   ![image](https://github.com/Fernando426/CapstoneN01/assets/107952724/2dccd592-df86-4878-b1ac-8e6c34be85c3)